tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -939,9 +942,8 @@ def test_ds_block_insertafter_regex(ansible_zos_module, dstype):
     hosts = ansible_zos_module
     ds_type = dstype
     params = dict(insertafter="ZOAU_ROOT=", block="ZOAU_ROOT=/mvsutil-develop_dsed\nZOAU_HOME=\\$ZOAU_ROOT\nZOAU_DIR=\\$ZOAU_ROOT", state="present")
-    test_name = "DST1"
-    temp_file = "/tmp/{0}".format(test_name)
-    ds_name = test_name.upper() + "." + ds_type
+    temp_file = "/tmp/{0}".format(inspect.stack()[0][3])
```

Comment:
oh messing with getting information from the stack in this tests I don't think is the way to go. Since is just used to create a temp file why not let tempfile name it instead?